### PR TITLE
Making ProjectGraphLoader work again

### PR DESCRIPTION
**Pull request by TheAngryByrd**

### CODE REVIEW

File: test/Ionide.ProjInfo.Tests/Tests.fs [hunk 1]
```diff
@@ -849,8 +849,10 @@ let testFCSmapManyProj toolsPath workspaceLoader (workspaceFactory: ToolsPath ->
 
         loader.Notifications.Add (function
             | WorkspaceProjectState.Loaded (po, knownProjects, _) -> pos <- Map.add po.ProjectFileName po pos)
-
-        let fcsPo = FCS.mapToFSharpProjectOptions parsed.Head parsed
+        let parsedC1 = parsed |> Seq.find(fun x ->
+            x.ProjectFileName.EndsWith(``sample3 Netsdk projs``.ProjectFile)
+        )
+        let fcsPo = FCS.mapToFSharpProjectOptions parsedC1 parsed
```
Comment (TheAngryByrd) [inline test/Ionide.ProjInfo.Tests/Tests.fs:855]: Can't guarantee order so have to find our console app

File: test/Ionide.ProjInfo.Tests/Tests.fs [hunk 2]
```diff
@@ -1241,13 +1243,21 @@ let debugTests toolsPath workspaceLoader (workspaceFactory: ToolsPath -> IWorksp
...
         let loader = workspaceFactory toolsPath
 
-        let parsed = loader.LoadProjects [ projPath ] |> Seq.toList
+        // let projPath = @"D:\Programowanie\Projekty\Ionide\dotnet-proj-info\src\Ionide.ProjInfo.Sln\Ionide.ProjInfo.Sln.csproj"
+        // let parsedProjs = loader.LoadProjects [ projPath ] |> Seq.toList
+
+        // printfn "%A" parsedProjs
 
-        printfn "%A" parsed
+        let slnPath = @"C:\Users\JimmyByrd\Documents\Repositories\public\TheAngryByrd\FsToolkit.ErrorHandling\FsToolkit.ErrorHandling.sln"
```
Comment (TheAngryByrd) [inline test/Ionide.ProjInfo.Tests/Tests.fs:1253]: Could revert

**Review by TheAngryByrd — COMMENTED** [review]
Self code review

File: test/Ionide.ProjInfo.Tests/Tests.fs (hunk 1 above)
Comment (baronfel) [reply test/Ionide.ProjInfo.Tests/Tests.fs:855]: that makes sense 👍

File: test/Ionide.ProjInfo.Tests/Tests.fs (hunk 2 above)
Comment (baronfel) [reply test/Ionide.ProjInfo.Tests/Tests.fs:1253]: eh, this test is a scratch-pad for folks anyway so I don't think the content is really special

**Review by baronfel — APPROVED** [review]
Thanks for this!

**Pull request merged**